src/client/config/thema.ts

```diff
@@ -140,6 +154,8 @@ export const DocumentTitles: DocumentTitlesConfig = {
   [AppRoutes['AFIS/FACTUREN']]: 'Lijst met facturen | Facturen en betalen',
   [AppRoutes['AFIS/BETAALVOORKEUREN']]:
     'Betaalvoorkeuren | Facturen en betalen',
+  '/passagiers-en-beroepsvaart/vergunningen/:kind/:page?': '',
+  '/passagiers-en-beroepsvaart/vergunning/:caseType/:id': '',
```

Comment:
Deze moge weg...

Reply:
Die waren er met een **quick fix** weer geplaats . Wanneer ik deze weer weghaal krijg ik heb een foutmelding bij DocumentTitles > Type '{ [x: string]: string | ((config: TrackingConfig) => "Home | Meldingen overzicht" | "Home | Dashboard" | "Inloggen | Mijn Amsterdam"); "/": (config: TrackingConfig) => "Home | Meldingen overzicht" | ... 1 more ... | "Inloggen | Mijn Amsterdam"; ... 56 more ...; "/facturen-en-betalen/betaalvoorkeuren": string; }' is missing the following properties from type 'DocumentTitlesConfig': "/passagiers-en-beroepsvaart/vergunningen/:kind/:page?", "/passagiers-en-beroepsvaart/vergunning/:caseType/:id"ts(2739)

Reply:
Check, ik heb hier een fix voor gemaakt. https://github.com/Amsterdam/mijn-amsterdam-frontend/pull/1642/commits/e39f6dd8eb09b71791cba53c4838bcda0396b737